cmd/tealdbg/local.go

```diff
@@ -335,7 +335,10 @@ func (r *LocalRunner) Setup(dp *DebugParams) (err error) {
 			r.runs[i].program = data
 			if IsTextFile(data) {
 				source := string(data)
-				ops, err := logic.AssembleStringWithVersion(source, r.proto.LogicSigVersion)
+				ops, err := logic.AssembleString(source)
```

Comment:
Old code allowed to control assembly version by setting`--proto` cli option. New code relies only on `#pragma version` statement.

Since `--proto` is not used by assembler anymore, this needs to be rephrased:
```go
debugCmd.Flags().StringVarP(&proto, "proto", "p", "", "Consensus protocol version for TEAL")
```
to something like "protocol version for TEAL evaluation".